content/en/docs/tasks/security/health-check/index.md

```diff
@@ -34,12 +34,10 @@ See the [authentication policy task](/docs/tasks/security/authn-policy/) for det
 
 ## Deploying Citadel with health checking
 
-To enable health checking, redeploy Citadel with helm:
+To enable health checking, redeploy Citadel:
 
 {{< text bash >}}
-$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
--x charts/security/templates/deployment.yaml \
---set global.mtls.enabled=true --set security.citadelHealthCheck=true > citadel-health-check.yaml
+$ istioctl manifest generate --set values.global.mtls.enabled=true --set values.security.citadelHealthCheck=true > citadel-health-check.yaml
```

Comment:
I suggest just using apply and drop kubectl apply below.

Reply:
The task uses the yaml file again later, so I left it as generate.
This doc, as well as many others, could certainly use some cleanup/mprovement, but I'll leave that for a future PR.